services/brig/src/Brig/API/Public.hs

```diff
@@ -381,47 +446,6 @@ sitemap o = do
   -- some APIs moved to servant
   -- end User Handle API
 
-  -- User Prekey API ----------------------------------------------------
-
-  post "/users/prekeys" (continue getMultiPrekeyBundlesH) $
-    jsonRequest @Public.UserClients
-      .&. accept "application" "json"
-  document "POST" "getMultiPrekeyBundles" $ do
-    Doc.summary
-      "Given a map of user IDs to client IDs return a \
-      \prekey for each one. You can't request information for more users than \
-      \maximum conversation size."
-    Doc.notes
-      "Prekeys of all clients of a multiple users. \
-      \The result is a map of maps, i.e. { UserId : { ClientId : Maybe Prekey } }"
-    Doc.body (Doc.ref Public.modelUserClients) $
-      Doc.description "JSON body"
-    Doc.response 200 "Prekey Bundles" Doc.end
-    Doc.errorResponse tooManyClients
```

Comment:
I don't see this in the servant endpoints, am I reviewing this too soon?

Reply:
It's refactored as the `GetMultiUserPrekeyBundleUnqualified` endpoint. For the qualified veresion (multiple domains) `GetMultiUserPrekeyBundleQualified` the path i mapped `POST /list-prekeys` which I don't like so much, but it's somewhat consistent with `list-users`.

Not ready yet, I'll re-request a review from you then.